src/components/HomePage/WhatOurStudentsSay.js

```diff
@@ -61,7 +61,7 @@ function WhatOurStudentsSay() {
 							className="p-6 mx-2.5 border rounded-lg"
 						>
 							<h1 className="h3-text mb-0">{eachReview.name}</h1>
```

Comment:
also if we can set the max height of the div so its doesn't change height based on the amount of text

Reply:
true, will get on that 😉

Reply:
done